canisters/trustworthy-metrics-canister/src/trustworthy-metrics-canister-backend/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+use ic_cdk_macros::*;
+use itertools::Itertools;
+use std::time::Duration;
+use types::{SubnetNodeMetricsArgs, SubnetNodeMetricsResponse};
+mod metrics_manager;
+mod stable_memory;
+mod types;
+
+const TIMER_INTERVAL_SEC: u64 = 60 * 60 * 6;
```

Comment:
Maybe add a comment explaining why is it this number? Is it at all related to the feature or is it a random pick 😄

Reply:
Metrics are recomputed daily -> changed to 24 hrs ;)